src/api/dependency-discovery/src/github.js

```diff
@@ -11,9 +10,10 @@ async function constructGitHubRequest(gitHubUrl) {
   const labels = ['hacktoberfest', 'good first issue', 'help wanted'];
 
   const [owner, repo] = new URL(gitHubUrl).pathname.substring(1).split('/');
+  const octokit = new Octokit();
```

Comment:
Can you add some plugins here like we do in `src/api/status/src/js/octokit.js` for throttling and abuse protection?

Reply:
I added the throttling here. I could not see if `octokit-plugin/retry` was included in [`octokit`](https://www.npmjs.com/package/octokit), but the `octokit-plugin/throttling` was in the documentation which leads me to believe `retry` is as well.

Started it all up and it seems to work, not sure how to test `throttling` or `retry` though